src/Storefront/Controller/PasswordPageController.php

```diff
@@ -44,7 +44,7 @@ public function showLogin(Request $request, SalesChannelContext $salesChannelCon
      */
     public function login(Request $request): Response
     {
-        $navigationId = $request->get('navigationId');
+        $navigationId = json_decode($request->query->get('redirectParameters'))->navigationId;
```

Comment:
seems much harder to read than before, why does the linter complain here?

Reply:
```
  Line   src/Storefront/Controller/PasswordPageController.php                   
 ------ ----------------------------------------------------------------------- 
  37     Call of internal method Symfony\Component\HttpFoundation\Request::get  
         Please refrain from using methods which are annotated with @internal   
         in the Shopware 6 repository.                                          
  47     Call of internal method Symfony\Component\HttpFoundation\Request::get  
         Please refrain from using methods which are annotated with @internal   
         in the Shopware 6 repository.                                          
 ------ ------------------------------------
```